recipes/nputop/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "nputop" %}
+{% set version = "0.0.4" %}
+{% set python_min = "3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/youyve/nputop/archive/v{{ version }}.tar.gz
+  sha256: c0650244af08d5006936738147f900c5de653cccd0972199306fd372f8e5b1d2
+
+build:
+  number: 0
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - nputop = nputop.cli:main
+  skip: True  # [not linux]  # 仅 Linux 构建
+
+requirements:
+  build:
+    - python {{ python_min }}
+    - setuptools
+  run:
+    - python  # 移除版本约束
```

```suggestion
    - python  >= {{ python_min }}
```

If you are using `noarch: python`, otherwise just `python`

---

I'm using non noarch packages, so I just removed the python version restrict.